benchmarks/accuracy/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+enable_testing()
```

Comment:
I would factor this up in the root `CMakeLists.txt`:

```cmake
if (BUILD_TESTS OR BUILD_BENCHMARKS)
    enable_testing()
endif()
```

I'm not sure if it's a detail or not: could calling `enable_testing()` twice cause problems? :thinking: